CHANGELOG.rst

```diff
@@ -1,3 +1,16 @@
+2.1.0 (2024-02-21)
+------------------
+
+* `#243 <https://github.com/pytest-dev/execnet/pull/243>`__: Added main_thread_only execmodel
+  which is derived from the thread execmodel and only executes ``remote_exec`` calls in the
+  main thread. Callers of ``remote_exec`` must use the returned channel to wait for a task to
```

Comment:
I think this is better in a separate paragraph. 👍 

```suggestion
  main thread. 
  
  Callers of ``remote_exec`` must use the returned channel to wait for a task to
```

Reply:
Applied, thanks!